recipes/pweave/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "0.25" %}
+{% set sha256 = "1c0f6921196646243eb7ff9eee742305909be2bc7a5eeeb06a7d1f66cc9758c7" %}
+
+package:
+  name: pweave
+  version: {{ version }}
+
+source:
+  fn: Pweave-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pweave/Pweave-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - Pweave = pweave.scripts:weave
+    - pweave = pweave.scripts:weave
+    - Ptangle = pweave.scripts:tangle
+    - ptangle = pweave.scripts:tangle
+    - pypublish = pweave.scripts:publish
+    - pweave-convert = pweave.scripts:convert
```

Comment:
Since these are defined in the setup.py entry_points, you don't have to redefine them here.  They transparently come through from setup.py and are available here.  You can remove lines 18-23 and keep the test commands section